src/main/java/com/google/cloud/tools/appengine/api/devserver/DefaultRunConfiguration.java

```diff
@@ -264,4 +266,13 @@ public File getDatastorePath() {
   public void setDatastorePath(File datastorePath) {
     this.datastorePath = datastorePath;
   }
+
+  @Override
+  public Map<String, String> getEnvironment() {
+    return environment;
+  }
+
+  public void setEnvironment(Map<String, String> environment) {
+    this.environment = environment;
```

Comment:
done. I also did this for the jvmArgs field (a List), for consistency. I don't think this allows to get rid of any other code though (since clients could presumable invoke CloudSdk#runDevAppServer1Command directly, in which case we still need a defensive copy here).